### DISCUSSION TITLE
Any plan to update firebase-database dependency?

### DISCUSSION THREAD

**Discussion by filipelinhares** [firebase/firebase-admin-node#1391] [General]
Hi Guys 😄 

The last _@firebase/database_ version has a bug fix to AppCheck. Do you have any plans to a release this update? I tried modifying my package-lock.json to enforce it but did not work as well.

Thanks in advance!

**Comment by lahirumaramba** [comment]
Hi @filipelinhares,
If you are referring to this [issue](https://stackoverflow.com/questions/68283347/how-to-implement-firebase-appcheck-inside-firebase-function-to-read-realtime-dat), we have discovered additional issues affecting it and the team is currently working on a fix. I will update here once a new `@firebase/database` version is ready with a fix. Thanks!

**Comment by lahirumaramba** [comment]
Thank you for your patience! The issue should be fixed in [`@firebase/database@0.10.8`](https://www.npmjs.com/package/@firebase/database). You might have to clean your node_modules or package-lock file to make sure you get the latest `@firebase/database@0.10.8` release. Let us know if you run into any issues.

**Reply by filipelinhares** [reply]
Thanks @lahirumaramba!
